src/components/Inputs/InputCheckbox.js

```diff
@@ -0,0 +1,39 @@
+export const InputCheckbox = props => {
+      const {options, value  } = props
+      const changeFeatureHandler = e => {
+      const value = e.target.value;
+      const isChecked = e.target.checked;
+  
+      if (isChecked) {
+        const newValue = [...props.value, value];
+        props.onChange(newValue);

```

Comment:
onChange also can be in destructuring and there you could use just "onChange(newValue)"